iothub/device/src/Transport/IDeviceIdentity.cs

```diff
@@ -3,15 +3,44 @@
 
 namespace Microsoft.Azure.Devices.Client.Transport
 {
+    /// <summary>
+    /// Interface for device configurations and common attributes.
+    /// </summary>
     internal interface IDeviceIdentity
```

Comment:
q - is this for device clients only and not for module clients?

Reply:
It appears so. Only in AMQP and Device connections only.